server/db/db.js

```diff
@@ -50,7 +50,7 @@ function createSensorsVitalFromRow(r, allLocations) {
   return new SensorsVital(r.id, r.missed_door_messages, r.is_door_battery_low, r.door_last_seen_at, r.reset_reason, r.state_transitions, r.created_at, r.is_tampered, location)
 }
 
-async function beginTransaction(retryCount = 0) {
+async function runBeginTransactionWithRetries(retryCount) {
```

Comment:
Does `retryCount` need to be initialized anywhere?

Reply:
Nope! Since retryCount is a parameter, it will get the value of whatever value is passed when the function is called.

The code before had `beginTransaction(retryCount = 0)` because I wanted to give `retryCount` a default value of `0` in the case of `beginTransaction` being called with no argument. But because we've changed code, giving `retryCount` a default value is not necessary.